sklearn/metrics/tests/test_classification.py

```diff
@@ -2542,6 +2542,19 @@ def test_log_loss():
     assert_almost_equal(loss, 1.0630345, decimal=6)
 
 
+def test_log_loss_eps_auto(global_dtype):
```

Comment:
If we're going out of our way to ensure compliance w.r.t `FP16` as well, perhaps we ought to include a separate test for that since `global_dtype` does not take on `np.float16`. It's unlikely to be broken if `np.float{32, 64}` work but it doesn't hurt to cover it.